### PR TITLE
Mate

### PR DESCRIPTION
**Pull request by pyhalov**

### CODE REVIEW

File: components/desktop/gnome3/yelp/patches/01-menu-entry.patch [hunk 1]
```diff
@@ -0,0 +1,10 @@
+--- yelp-3.20.1/yelp.desktop.in.in.~1~	2016-05-24 12:33:26.850149207 +0300
++++ yelp-3.20.1/yelp.desktop.in.in	2016-05-24 12:34:23.691695600 +0300
+@@ -1,6 +1,6 @@
+ [Desktop Entry]
+ _Name=Help
+-_Comment=Get help with GNOME
++_Comment=_Comment=Browse and search GNOME and OpenSolaris documentation
```
Comment (xen0l) [inline components/desktop/gnome3/yelp/patches/01-menu-entry.patch]: Shouldn't this be OpenIndiana documentation?

Comment (pyhalov) [inline components/desktop/gnome3/yelp/patches/01-menu-entry.patch]: Seems so

File: components/desktop/gnome3/yelp/patches/03-yelp-groff.patch [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+ 	-DYELP_ICON_PATH=\"$(YELP_ICON_PATH)\" \
+ 	-DYELP_WEB_EXTENSIONS_DIR=\""$(pkglibdir)/"web-extensions\" \
+ 	-I$(top_builddir)/libyelp
+diff --git a/libyelp/yelp-groff b/libyelp/yelp-groff
+new file mode 100755
+index 0000000..5348024
+--- /dev/null
++++ b/libyelp/yelp-groff
+@@ -0,0 +1,49 @@
++#!/usr/bin/sh
```
Comment (xen0l) [inline components/desktop/gnome3/yelp/patches/03-yelp-groff.patch:53]: Maybe use /usr/bin/ksh, so we aren't suprised if by accident /usr/bin/sh switches to a different shell? It shall happen in a fork.

Comment (pyhalov) [inline components/desktop/gnome3/yelp/patches/03-yelp-groff.patch:53]: script works in bash/ksh... If it doesn't work in dash, we'll have to fix it. When they switch sh to dash we'll have a lot of fun anyway.

**Pull request merged**